tests/ci/run_fips_tests.sh

```diff
@@ -35,7 +36,7 @@ if static_linux_supported || static_openbsd_supported; then
   run_build -DFIPS=1 \
     -DCMAKE_C_FLAGS="-DBORINGSSL_FIPS_BREAK_TESTS -DAWSLC_FIPS_FAILURE_CALLBACK" \
     -DCMAKE_CXX_FLAGS="-DAWSLC_FIPS_FAILURE_CALLBACK"
-  ./test_build_dir/crypto/crypto_test
+  shard_gtest ./test_build_dir/crypto/crypto_test
```

Comment:
why is this needed?

Reply:
I just noticed it was slow and Torben reminded it exists. It could go in it's own review.

Reply:
no, seems fine.